pke/kyber/internal/common/field_test.go

```diff
@@ -35,9 +36,29 @@ func TestBarrettReduceFull(t *testing.T) {
 	}
 }
 
+func randSliceUint32Max(N uint, max uint32) []uint32 {
```

Comment:
nit: s/N/length/ in order to match the reimplementation of this function below.

Reply:
resolved